src/custom/state/ethFlow/atoms.ts

```diff
@@ -0,0 +1,8 @@
+import { selectAtom } from 'jotai/utils'
+import { atomWithStorage } from './atoms/utils'
+
+/**
+ * Base atom that stores the user's set native ETH sell value
+ */
+export const userNativeEthFlow = atomWithStorage<boolean>('userNativeEthFlow', true)
+export const isUserNativeEthFlow = selectAtom(userNativeEthFlow, (value) => value)
```

Comment:
You don't need that AFAICT.
You can access the `userNativeEthFlow` atom directly.

If you only want to read it, you can use https://jotai.org/docs/utils/use-atom-value
But I see you already uses that so the change should be minimal